app/main.py

```diff
@@ -1,2 +1,49 @@
+class Node:
+    def __init__(self, key: object, value: object) -> None:
```

Comment:
This class better be defined inside of a Dictionary since it is a part of its implementation and will not be used outside of it.